src/formAdd.js

```diff
@@ -0,0 +1,44 @@
+// Import modules
+import Leader from './leaderObject.js';
+import Storage from './localStorage.js';
+import UI, { array } from './domObject.js';
+
+// Grab necessary elements from HTML
+const leaderName = document.querySelector('.leader-name');
+const leaderScore = document.querySelector('.leader-score');
+export const uiObject = new UI(); // object to be displayed in the DOM
+export let leadersArray = Storage.getStorage(); // eslint-disable-line import/no-mutable-exports
```

Comment:
- [ ] **[OPTIONAL]:** Although this was not required in this milestone, please remember to avoid disabling the linters, they ensure a consistent format and best practices across your source files! 🤓 🛠️